core/schema/directory.graphqls

```diff
@@ -190,6 +190,9 @@ type Directory {
 
     "Target build stage to build."
     target: String
+
+    "Secrets to pass to the build."
+    secrets: [SecretID!]
```

Comment:
Maybe add a small sentence about the `/run/secrets` so the user knows where the secret will be from the doc.

Reply:
Very good point, thanks.